hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientScheduledFutureProxy.java

```diff
@@ -186,7 +186,7 @@ public boolean isDone() {
     @Override
     public V get()
             throws InterruptedException, ExecutionException {
-        return this.get0().join();
+        return this.get0().get();
```

Comment:
`join` peels exceptions wrapped in `ExecutionException`s, so it would unwrap the exception in the received `ExecutionException`.